src/components/seo.js

```diff
@@ -1,93 +1,74 @@
+  const seo = {
+    title: title || defaultTitle,
+    description: description || defaultDescription,
+    image: `${siteUrl}${image || siteMetadataImage}`,
+    url: `${siteUrl}${pathname || ""}`,
+    twitterUsername
+  };
 
   return (
-    <Helmet
-      htmlAttributes={{
-        lang,
-      }}
-      title={title}
-      titleTemplate={`${title} | ${site.siteMetadata.title}`}
-      link={
-        canonical
-          ? [{ rel: "canonical", key: canonical, href: canonical }]
-          : []
-      }
-      meta={[
-        {
-          name: "description",
-          content: metaDescription,
-        },
-        {
-          property: "og:title",
-          content: title,
-        },
-        {
-          property: "og:description",
-          content: metaDescription,
-        },
-        {
-          property: "og:url",
-          content: url,
-        },
-        {
-          property: "og:type",
-          content: "website",
-        },
-        {
-          property: "og:image",
-          content: metaImage,
-        },
-        {
-          name: "twitter:card",
-          content: "summary_large_image",
-        },
-        {
-          name: "twitter:creator",
-          content: site.siteMetadata.twitterUsername,
-        },
-        {
-          name: "twitter:title",
-          content: title,
-        },
-        {
-          name: "twitter:description",
-          content: metaDescription,
-        },{
-          name: "twitter:image",
-          content: metaImage,
-        },
-      ].concat(meta)}
-    >
+    <>
+      <title>{seo.title}</title>
+      <html lang="en" />
+      <meta name="description" content={seo.description} />
+      <meta name="image" content={seo.image} />
+      <meta name="og:title" content={seo.title} />
+      <meta name="og:description" content={seo.description} />
+      <meta name="og:url" content={seo.url} />
+      <meta name="og:type" content="website" />
+      <meta name="og:image" content={seo.image} />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:title" content={seo.title} />
+      <meta name="twitter:url" content={seo.url} />
+      <meta name="twitter:description" content={seo.description} />
+      <meta name="twitter:image" content={seo.image} />
+      <meta name="twitter:creator" content={seo.twitterUsername} />
+      <meta charSet="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <link rel="shortcut icon" type="image/x-icon" href={FavIcon} />
+      <link rel="preconnect" href="https://fonts.gstatic.com/" />
+      <link
+        rel="preload"
+        href="https://fonts.googleapis.com/css?family=Open%20Sans:300,400,500,600,700,800&display=swap"
+        media="print"
+        onLoad="this.media='all'"
+        as="font"
+        crossOrigin
```

Comment:
Add the `noscript` code as well.
Secondly, the use of `stylesheet` instead of `font` was intentional for performance improvement hacks, and I think it is causing the increase in TTB to some extent. Do test that out to confirm it.

Reply:
here on this google blog https://web.dev/codelab-preload-web-fonts/ they mentioned to use as="font". I'll try both and see which one is best.